app/src/main/java/com/sun/premierleague/ui/team/TeamContact.kt

```diff
@@ -0,0 +1,23 @@
+package com.sun.premierleague.ui.team
+
+import com.sun.premierleague.base.BasePresenter
+import com.sun.premierleague.base.BaseView
+import com.sun.premierleague.data.model.Coache
+import com.sun.premierleague.data.model.Player
+import com.sun.premierleague.data.model.TeamItem
+
+interface TeamContact {
+    interface View : BaseView {
+        fun showTeam(data: List<TeamItem>)
```

Comment:
Đặt tên clear hơn giúp đoạn code dễ hiểu hơn 
```suggestion
        fun showTeam(teams: List<TeamItem>)
```